account_statement_import_camt/models/parser.py

```diff
@@ -217,6 +219,8 @@ def parse_statement(self, ns, node):
             result["date"] = sorted(
                 transactions, key=lambda x: x["date"], reverse=True
             )[0]["date"]
+        for seq, vals in enumerate(transactions, start=1):
+            vals["sequence"] = seq
```

Comment:
This looks generic. Is this something that should be added to account_bank_import instead, for any imported statements with transactions that do not contain a sequence number?